pymongo/message.py

```diff
@@ -1503,7 +1504,7 @@ class _OpReply(object):
 
     def __init__(self, flags, cursor_id, number_returned, documents):
         self.flags = flags
-        self.cursor_id = cursor_id
+        self.cursor_id = Int64(cursor_id)
```

Comment:
See https://jira.mongodb.org/browse/DRIVERS-1850.

When we decode a OP_REPLY cursor_id we decode it to a regular python int (with unlimited precision).
When we decode a command response the cursor.id field is decoded to a special bson.Int64 class.
So on <=3.0 the cursor id isn’t a “long”.

This change makes the cursor id always be Int64 regardless of the server version.

Reply:
Nice discovery.